zerver/models.py

```diff
@@ -1025,6 +1025,9 @@ class UserProfile(AbstractBaseUser, PermissionsMixin):
     # Hours to wait before sending another email to a user
     EMAIL_REMINDER_WAITPERIOD = 24
 
+    # Whether or not the user wants to sync their drafts.
+    drafts_syncing_enabled = models.BooleanField(default=False)
```

Comment:
I think we should call this `enable_drafts_synchronization`, and default it to True; I imagine it'll be rare that folks turn this off, and we'll save a migration in the future.